### DISCUSSION TITLE
Is there a setPixel method like in Unity.

### DISCUSSION THREAD

**Discussion by zver322** [stride3d/stride#1606] [Q&A] [answered]
Hello everyone,
I tried tofidure out how can I edit a specific pixel of an existing texture. I know only one way how I can change texture and it is a setData method. And as I understand  it can only rewrite a texture, but I want to edit and set a specific pixel.
I've just wanted to write my own drawLine method for Stride. It modifies texture and I need to use setPixel method as in Unity, but I don't know it there such staff in Stride 
Does anyone know any way to do such thing?

**Comment by NicusorN5** [comment, marked as answer]
You can write your own `SetPixel` method if `SetData()` already exists.

I'd advise against that since rewritting data **multiple times per frame** for a single dynamic buffer will result in a major performance penalty.

**Reply by zver322** [reply]
Thank you, it's really useful advice. I will try to do it.
